### ISSUE TITLE
Make tests run in parallel

### ISSUE THREAD

**Issue by warpech** [Starcounter/UniformDocs#378]
This project has 191 tests, which are NOT set to run in parallel. BrowserStack has potential to run them in parallel, but the project needs to be prepared for it.

It is simple to turn any test in NUnit to be parallel, but we need to check if none of the tests in this project are stateful. The tests that utilize data binding and statefulness might affect other tests that run in parallel.

This is to be done after: https://github.com/Starcounter/RebelsLounge/issues/210

**Comment by warpech** [comment]
Closing in favor of #396
